docs/source/en/api/pipelines/fabric.md

```diff
@@ -0,0 +1,84 @@
+
+The original codebase can be found here: 
+- *FABRIC*: [sd-fabric/fabric](https://github.com/sd-fabric/fabric)
+
+Available Checkpoints are:
+- *dreamlike-photoreal-2.0* [dreamlike-art/dreamlike-photoreal-2.0](https://huggingface.co/dreamlike-art/dreamlike-photoreal-2.0)
+
+
+## Usage Example
+
+Before using Fabric make sure to have `transformers`, `accelerate`, `huggingface_hub`  installed. 
+You can install the libraries as follows:
+
+```
+pip install transformers
+pip install accelerate
+pip install huggingface_hub
+```
+
+### Text-to-Image
+
+You can use Fabric as follows for *text-to-image*:
+
+```py
+from diffusers import FabricPipeline
+import torch
+
+model_id = "dreamlike-art/dreamlike-photoreal-2.0"
+pipe = StableDiffusionPipeline.from_pretrained(model_id, torch_dtype=torch.float16)
+pipe = pipe.to("cuda")
+
+prompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k"
+neg_prompt = "bad anatomy, cropped, lowres"
+image = pipe(prompt=prompt, negative_prompt=neg_prompt).images[0]
+```
+
+You can use Fabric as follows for *text-to-image-with-feedback*:
+
+```py
+from diffusers import FabricPipeline
+import torch
+
+model_id = "dreamlike-art/dreamlike-photoreal-2.0"
+pipe = StableDiffusionPipeline.from_pretrained(model_id, torch_dtype=torch.float16)
+pipe = pipe.to("cuda")
+
+prompt = "Astronaut in a jungle, cold color palette, muted colors, detailed, 8k"
+neg_prompt = "bad anatomy, cropped, lowres"
+liked = ["path/to/image"]
+disliked = ["path/to/image"]
+image = pipe(prompt=prompt, negative_prompt=neg_prompt,liked=liked,disliked=disliked).images[0]
+```
+
+Let's have a look at the images (*512X512*)
+
+| Without Feedback            | With Feedback  (1st image)          |
+|---------------------|---------------------|
+| ![Image 1](https://drive.google.com/uc?export=view&id=12wxbikt7834eRTK40legR5PtJmFLNH34) | ![Feedback Image 1](https://drive.google.com/uc?export=view&id=1YcFPDHSRr2OE3hy-5lvr8An21Jum85D5) | 
+
```

Comment:
Let's add a Tip here to let users know they can switch/play around with different schedulers:
```
<Tip>

Make sure to check out the Stable Diffusion [Tips](overview#tips) section to learn how to explore the tradeoff between scheduler speed and quality, and how to reuse pipeline components efficiently!

</Tip>
```